src/Raven.Server/ServerWide/ClusterStateMachine.License.cs

```diff
@@ -29,24 +31,33 @@ public sealed partial class ClusterStateMachine
 {
     private const int MinBuildVersion60000 = 60_000;
     private const int MinBuildVersion60102 = 60_102;
+    private const int MinBuildVersion60105 = 60_105;
```

Comment:
This is incorrect. 6.0.105 build number is 60039:
![image](https://github.com/user-attachments/assets/c8b86ced-6851-41c7-954c-95b7e1f9938c)

Reply:
Done